fusion/model.py

```diff
@@ -0,0 +1,189 @@
+"""Model for fusion class"""
+import json
+import re
+from pydantic import BaseModel, validator, ValidationError, StrictStr, \
+    StrictInt, StrictBool
+from typing import Optional, List, Union
+
+
+class GeneDescriptor(BaseModel):
```

Comment:
Should also add the other fields that a GeneDescriptor can have: https://vrsatile.readthedocs.io/en/latest/value_object_descriptor/vod_index.html#gene-descriptor

Reply:
Ideally, we are capturing (perhaps direct importing?) the same GeneDescriptor class, but are only using a few components in implementation / instantiation of this class in the gene fusion project.